extensions/tekton/generator-v1beta1/go.mod

```diff
@@ -2,10 +2,17 @@ module github.com/fabric8io/kubernetes-client/extensions/tekton/generator
 
 require (
 	github.com/fabric8io/kubernetes-client/generator v0.0.0
-	github.com/tektoncd/pipeline v0.11.2
-	k8s.io/apimachinery v0.17.2
+	github.com/tektoncd/pipeline v0.12.1
+	k8s.io/api v0.17.3 // indirect
+	k8s.io/apimachinery v0.17.3
+	k8s.io/client-go v0.17.3 // indirect
 )
 
-replace github.com/fabric8io/kubernetes-client/generator v0.0.0 => ./../../../generator
+replace (
+	github.com/fabric8io/kubernetes-client/generator v0.0.0 => ./../../../generator
+	k8s.io/api => k8s.io/api v0.16.5
+	k8s.io/apimachinery => k8s.io/apimachinery v0.16.5
+	k8s.io/client-go => k8s.io/client-go v0.16.5
```

Comment:
Do you know why this is necessary?

Reply:
I did this to match with whatever is used in upstream https://github.com/tektoncd/pipeline/blob/release-v0.12.x/go.mod#L63